src/tests/electron/common/view-controllers/log-controller.ts

```diff
@@ -33,4 +36,13 @@ export class LogController {
     public resetServerLog(currentContext: string): void {
         fs.unlinkSync(generateServerLogPath(this.getOutputLogsDir(currentContext)));
     }
+
+    public async waitForAdbLogToContain(
```

Comment:
Should there also be a `waitForServerLogToContain` method?

Reply:
I wrote both methods at first and then decided to wait and see if we needed one for server logging. It'll be trivial to add it back if we do.